src/content/pages/privacy-guides/vpn-latency-explained.md

```diff
@@ -25,20 +25,20 @@ It characterizes how a user feels the responsiveness when doing something over t
 
 Why is there any latency in the first place? Computers are fast and get faster every year after all. Most of the latency occurring is due to the distance [a network packet](https://www.techtarget.com/searchnetworking/definition/packet) has to travel.
 
-In fact, our fastest communication transport medium is the optical fiber. It can only carry data at two third of light speed, as the light beam inside the fiber is bouncing with a specific angle instead of going in a straight line. Light travels at approximately 300 000 kilometers per second, this is fast, but not enough to avoid latency.
+In fact, our fastest communication transport medium is the optical fibre. It can only carry data at two third of light speed, as the light beam inside the fibre is bouncing with a specific angle instead of going in a straight line. Light travels at approximately 300 000 kilometres per second, this is fast, but not enough to avoid latency.
```

Comment:
it seems a US spelling to UK spelling change right? We want to stick with the US words and keep consistent with this choice.